packages/example/src/pages/SplitLayout.tsx

```diff
@@ -1,15 +1,20 @@
-import React, { FC, useState } from "react";
+import React, { FC, useCallback, useState } from "react";
 import { ThemeProvider } from 'styled-components';
 import { GlobalUI, defaultTheme, useThemeToggle, ContentLayout, SplitLayout, FlexContentPlaceholder } from "scorer-ui-kit";
 import ExamplesFilename from '../components/ExamplesFilename';
 
 const SplitLayouts: FC = () => {
 
   const {onThemeToggle, isLightMode} = useThemeToggle();
-  
+
   const [layout] = useState<'horizontal'|'vertical'>('horizontal');
   const [reverse] = useState<boolean>(false);
-  
+
+
+  const sideStateCallback = useCallback((currentState) => {
+    console.log(`Side Area Current State ${currentState}`);
```

Comment:
Just as a small readability tweak, it would be good to have a `:` in there, like this...

console.log(`Side Area Current State: ${currentState}`);